backend/platform_controls_router.py

```
@@ -0,0 +1,106 @@
+
+    def _admin(user: dict) -> None:
+        from backend.company_api import _is_admin
+
+        if not _is_admin(user):
+            raise HTTPException(
+                status_code=403,
+                detail="Admin role required to view or change platform controls",
+            )
+
+    @router.get("")
+    async def list_controls(user: dict = Depends(get_current_user)) -> dict[str, Any]:
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)